pytensor/configparser.py

```diff
@@ -68,7 +69,123 @@ def __exit__(self, *args):
+    assert_no_cpu_op: str
+    unpickle_function: bool
+    # add_compile_configvars
+    mode: str
+    cxx: str
+    linker: str
+    allow_gc: bool
+    optimizer: str
+    optimizer_verbose: bool
+    on_opt_error: str
+    nocleanup: bool
+    on_unused_import: str
+    gcc__cxxflags: str
+    cmodule__warn_no_version: bool
+    cmodule__remove_gxx_opt: bool
+    cmodule__compilation_warning: bool
+    cmodule__preload_cache: bool
+    cmodule__age_thresh_use: int
+    cmodule__debug: bool
+    compile__wait: int
+    compile__timeout: int
+    ctc__root: str
+    # add_tensor_configvars
+    tensor__cmp_sloppy: int
+    lib__amblibm: bool
+    tensor__insert_inplace_optimizer_validate_nb: int
+    # add_traceback_configvars
+    traceback__limit: int
+    traceback__compile_limit: int
+    # add_experimental_configvars
+    # add_error_and_warning_configvars
+    warn__ignore_bug_before: int
+    exception_verbosity: str
+    # add_testvalue_and_checking_configvars
+    print_test_value: bool
+    compute_test_value: str
+    compute_test_value_opt: str
+    check_input: bool
+    NanGuardMode__nan_is_error: bool
+    NanGuardMode__inf_is_error: bool
+    NanGuardMode__big_is_error: bool
+    NanGuardMode__action: str
+    DebugMode__patience: int
+    DebugMode__check_c: bool
+    DebugMode__check_py: bool
+    DebugMode__check_finite: bool
+    DebugMode__check_strides: int
+    DebugMode__warn_input_not_reused: bool
+    DebugMode__check_preallocated_output: str
+    DebugMode__check_preallocated_output_ndim: int
+    profiling__time_thunks: bool
+    profiling__n_apply: int
+    profiling__n_ops: int
+    profiling__output_line_width: int
+    profiling__min_memory_size: int
+    profiling__min_peak_memory: bool
+    profiling__destination: str
+    profiling__debugprint: bool
+    profiling__ignore_first_call: bool
+    on_shape_error: str
+    # add_multiprocessing_configvars
+    openmp: bool
+    openmp_elemwise_minsize: int
+    # add_optimizer_configvars
+    optimizer_excluding: str
+    optimizer_including: str
+    optimizer_requiring: str
+    optdb__position_cutoff: float
+    optdb__max_use_ratio: float
+    cycle_detection: str
+    check_stack_trace: str
+    metaopt__verbose: int
+    metaopt__optimizer_excluding: str
+    metaopt__optimizer_including: str
+    # add_vm_configvars
+    profile: bool
+    profile_optimizer: bool
+    profile_memory: bool
+    vm__lazy: bool | None
+    # add_deprecated_configvars
+    unittests__rseed: str
+    warn__round: bool
+    # add_scan_configvars
+    scan__allow_gc: bool
+    scan__allow_output_prealloc: bool
+    # add_numba_configvars
+    numba__vectorize_target: str
+    numba__fastmath: bool
+    numba__cache: bool
+    # add_caching_dir_configvars
+    compiledir_format: str
+    base_compiledir: Path
+    compiledir: Path
+    # add_blas_configvars
+    blas__ldflags: str
+    blas__check_openmp: bool
+    # add CUDA (?)
+    cuda__root: Path | None
```

Comment:
Many of these are no longer relevant and should be removed. Checking if they're used anywhere in the codebase should clarify what's what